src/Workspaces/Core/Portable/CodeFixesAndRefactorings/FixAllKind.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.CodeFixesAndRefactorings
+{
+    /// <summary>
+    /// An enum to distinguish if we are performing a Fix all occurences for a code fix or a code refactoring.
```

Comment:
when do we care or need to know this?

Reply:
This is used at a few places where we want to share code between code fixes and refactorings and make leaf level choices, such as function ID for telemetry.

Reply:
dont' love it.  but if you think it's important, then ok :)